mapstory/apps/thumbnails/tasks.py

```diff
@@ -229,7 +255,7 @@ def run(self, pk, overwrite=False):
             print "EXCEPTION - thumbnail generation"
             print(e)
             print traceback.format_exc()
-            raise
+            self.retry(max_retries=5, countdown=31) # retry in 31 seconds (auth cache timeout)
```

Comment:
Does the auth caching problem always take 30 seconds, or is that the worst case?
If it's the latter, we could probably try it every 5s or something and just have more retries. (unless hitting it early causes problems)

Reply:
I appears to always take 30 seconds (although its no longer occurring).  The only errors that should be happening now are things like the not being able to get basemap images.  
I might be able to set this to 5 seconds with more retries and exponential delay.